### PR TITLE
Add icon-shield-check

### PR DESCRIPTION
**Pull request by cbfrance**

### CODE REVIEW

File: .changeset/fresh-pans-cry.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@cypress-design/icon-registry': minor
```
Comment (elevatebart) [inline .changeset/fresh-pans-cry.md:2]:
```suggestion
'@cypress-design/icon-registry': minor
'@cypress-design/vue-icon': minor
'@cypress-design/react-icon': minor
```

**Review by elevatebart — CHANGES_REQUESTED** [review]
small changeset update

File: .changeset/fresh-pans-cry.md (hunk 1 above)
Comment (ryanjwilke) [reply .changeset/fresh-pans-cry.md:2]: @cbfrance Agreed with this point ☝️ 

We should fix this before releasing to make sure we bump the version of each component.

**Review by ryanjwilke — CHANGES_REQUESTED** [review]
Requested changes for one small issue, otherwise the SVGs look good.

**Review by elevatebart — APPROVED** [review]

**Pull request merged**